gradio/interface.py

```diff
@@ -274,7 +274,7 @@ def __init__(
         else:
             raise ValueError("Invalid value for parameter: interpretation")
 
-        self.api_mode = False
+        self.api_mode = kwargs.get("_api_mode", False)
```

Comment:
Would it be better to include `_api_mode` as an explicit but undocumented parameter in `Interface` with a default value of `False`? Better to have all of the parameters and their defaults in one place

Reply:
Yea it's better to be explicit. I just didn't want to add something resembling `api_mode` to the signature since it was removed previously. Just pushed this up and will merge! 

Thank you for the review.